examples/minimal/main.tf

```diff
@@ -18,8 +18,7 @@ module "api_gateway" {
 }
 
 module "api_gateway_stage" {
-  source  = "terraform.registry.launch.nttdata.com/module_collection/api_gateway_v2_stage/aws"
-  version = "~> 1.0"
+  source = "../.."
```

Comment:
Is this source REF expected? 

I don't remember seeing something like this in previous examples.